src/registrar/views/domain.py

```diff
@@ -791,10 +791,11 @@ class DomainInvitationDeleteView(DomainInvitationPermissionDeleteView, SuccessMe
     object: DomainInvitation  # workaround for type mismatch in DeleteView
 
     def get_success_url(self):
+        messages.success(self.request, self.get_success_message())
         return reverse("domain-users", kwargs={"pk": self.object.domain.id})
 
-    def get_success_message(self, cleaned_data):
-        return f"Successfully canceled invitation for {self.object.email}."
+    def get_success_message(self):
```

Comment:
**question (blocking)**: The `SuccessMiessageMixin` class uses the method with this name https://docs.djangoproject.com/en/5.0/ref/contrib/messages/#django.contrib.messages.views.SuccessMessageMixin and signature. Why did we need to change this?

Reply:
The mixin should be removed, I just forgot to do so.

There is a [documented bug](https://code.djangoproject.com/ticket/33587) (there are a few other examples of this, but that link is the most relevant) that SuccessMessageMixin does not work with BaseDeleteView. This is because there is no super call within BaseDeleteView.

I took a look at this again and I found that just swapping the of `DomainInvitationPermissionDeleteView, SuccessMessageMixin` " to `SuccessMessageMixin, DomainInvitationPermissionDeleteView`. Thanks for bringing it to my attention